index.html

```diff
@@ -6,40 +6,59 @@
+        #overlay {
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 100%;
+            color: #FFFFFF;
+            padding: 0 1em;
+        }
+    </style>
 </head>
 
 <body>
-    <div>
-        <h1>Prototype Viewer MediaWiki 3D GLTF Data, STL data is already available </h1>
-        <select id="selection" onchange="loadSelection()">
-            <option value="">Please choose a model</option>
-            <option value="3Ddata.glb">GLB Lion Cologne Zoo - https://codeforcologne.github.io/Denkmal-4D-Koeln/
-            </option>
-            <option value="stl_lion.stl">STL Lion Cologne Zoo - https://codeforcologne.github.io/Denkmal-4D-Koeln/
-            </option>
-            <option value="ganesha.glb">GLB Ganesha Cologne Zoo - https://codeforcologne.github.io/Denkmal-4D-Koeln/
-            </option>
-            <option value="ganesha_stl.stl">STL Ganesha Cologne Zoo - https://codeforcologne.github.io/Denkmal-4D-Koeln/
-            </option>
-            <option value="3D_Model_Hydria.stl">STL Hydria -https://commons.wikimedia.org/wiki/File:3D_Model_Hydria.stl
-            </option>
-            <option value="3D_model_of_an_African_elephant.stl">STL African_elephant
-                -https://commons.wikimedia.org/wiki/File:3D_model_of_an_African_elephant.stl</option>
-            <option value="Avocado.glb">GLB Avocado - KhronosGroup Models Testing</option>
-            <option value="BarramundiFish.glb">GLB BarramundiFish - KhronosGroup Models Testing</option>
-            <option value="LightsPunctualLamp.glb">LightsPunctualLamp - KhronosGroup Models Testing</option>
-            <option value="Duck.glb">GLB Duck - KhronosGroup Models Testing</option>
-            <option value="NodePerformanceTest.glb">Wait a little bit! NodePerformanceTest - KhronosGroup Models Testing
-            </option>
-            <option value="NodePerformanceTest_opt.glb">Wait a little bit! NodePerformanceTest - Optimized with
-                gltf-transform -
-                KhronosGroup Models Testing
-            </option>
-            <option value="grinkopf.glb">Wait a little bit! Grinkopf Cologne -
-                https://codeforcologne.github.io/Denkmal-4D-Koeln/</option>
-            <option value="grinkopf_opt.glb">Grinkopf Cologne - Optimized with gltf-transform
-                https://codeforcologne.github.io/Denkmal-4D-Koeln/</option>
-            <option value="coffeemat.glb">GLB coffeemat with KTX2 cpmpression - KhronosGroup Models Testing</option>
+    <div id="overlay">
+        <h1>Prototype Viewer MediaWiki 3D</h1>
+        <p>GLTF and STL formats are supported</p>
+        <select id="selection" onchange="loadSelection()" placeholder="Please choose a model">
+            <option value="" disabled selected>Please choose a model</option>
+            <optgroup label="STL">
+                <option value="stl_lion.stl">Lion Cologne Zoo - https://codeforcologne.github.io/Denkmal-4D-Koeln/
```

Comment:
I just grouped the options by file format, found it a bit easier to hop between models quickly this way.